gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -0,0 +1,122 @@
+require 'postgres_ha_admin/failover_databases'
+require 'postgres_ha_admin/database_yml'
+require 'pg'
+require 'linux_admin'
+
+module PostgresHaAdmin
+  class FailoverMonitor
+    FAILOVER_ATTEMPTS = 60
```

Comment:
This seems a bit high... but we can also tweak that when we look into how we want to set the repmgrd parameters that govern how long the failover process will actually take.